conf/defaults.ini

```diff
@@ -948,7 +948,7 @@ session_duration = "15m"
 
 # Set the plugins that will receive AWS settings for each request (via plugin context)
```

Comment:
Is "aws settings" everything in the `aws` section of the grafana config?

Reply:
yup, all the other fields